tox.ini

```diff
@@ -7,9 +7,12 @@ envlist =
     py35,
     py36,
     py36-datetime,
+    py37,
+    pypy,
+    pypy3,
```

Comment:
PyPy should get removed here, too to prevent errors on the tox runs of the developers.